packages/old-import/src/widgets/options.ts

```diff
@@ -118,6 +118,7 @@ const optionMapping: OptionMapping = {
     enableRtl: (oldOptions) => oldOptions.enableRtl,
     maximumAmountPosts: (oldOptions) => oldOptions.maximumAmountOfPosts,
     textLinesClamp: (oldOptions) => oldOptions.textLinesClamp,
+    hideDescription: (oldOptions) => oldOptions.hideDescription ?? false,
```

Comment:
So we map it to undefined here which will then use the current default value of our rss widget definition